src/assets/styles/header.scss

```diff
@@ -0,0 +1,279 @@
+.header {
+  background: $dark;
+  text-align: center;
+  padding: 1em;
+}
+
+.header-inner {
+  margin: 0 auto;
+  width: 80%;
+  max-width: 500px;
+}
+
+.toolbar {
+  position: relative;
+}
+
+.toolbar.-desktop {
+  display: none;
+}
+
+.logo {
+  max-width: 210px;
+}
+
+.cta {
+  font-family: "Freight Sans CMP";
+  color: $white;
+  font-size: 2em;
+  margin: .1em 0;
+}
+
+.search-wrap {
+  height: 2em;
+  overflow: auto;
+  input,
+  button {
+    height: 100%;
+    display: block;
+    float: left;
+    padding: 0;
+    margin: 0;
+    border: 0;
+    box-sizing: border-box;
+  }
+  input {
+    width: 80%;
+  }
+  button {
+    width: 20%;
+    background: $dark-red;
+    &:hover {
+      background: darken($dark-red, 5%);
+    }
+  }
+}
+
+.button-group {
+  margin-top: $offset;
+  overflow: auto;
+  button {
+    width: calc(50% - #{$offset / 2});
+    float: left;
+    &:last-child {
+      float: right;
+    }
+  }
+}
+
+.filter-events {
+  position: fixed;
+  z-index: 9999;
+  left: 0;
+  right: 0;
+  // NOTE: extreme magic number.
+  top: 82px;
```

Comment:
It would be better to get this value by reading the height of some actual elements and doing some math